clase-6/tarea2.js

```diff
@@ -0,0 +1,75 @@
+// TAREA:
+// Crear una interfaz que permita agregar ó quitar (botones agregar y quitar) inputs+labels para completar el salario anual de cada integrante de la familia que trabaje.
+// Al hacer click en "calcular", mostrar en un elemento pre-existente el mayor salario anual, menor salario anual, salario anual promedio y salario mensual promedio.
+// Punto bonus: si hay inputs vacíos, ignorarlos en el cálculo (no contarlos como 0).
+
+const $numeroIntegrantes = document.querySelector("#integrantes-trabajo");
+const $botonAgregar = document.querySelector("#agregar");
+const $integrantesContainer = document.querySelector("#integrantes");
+const $botonCalcular = document.querySelector("#calcular");
+const $botonReiniciar = document.querySelector("#reiniciar");
+
+function crearIntegrante() {
+  let nuevoIntegrante = $numeroIntegrantes.value;
+  let numeroNuevoIntegrante;
+  
+  for(let i = 1; i <= nuevoIntegrante; i++) {
+    numeroNuevoIntegrante = i;
```

Comment:
no te conviene modificar la variable que estás reocrriendo en el for. Para eso ya tenés `i`

i <= cantidadIntegrates tendría más sentido, y luego Salario anual integrante # i